src/cluster.ml

```diff
@@ -182,6 +184,7 @@ let pull_and_serve multi_hash = function
   | `Ocamlorg_images name -> pull_and_serve (module Ocamlorg_images) ~name `Service multi_hash
   | `OCamlorg_v3b name -> pull_and_serve (module V3b_docker) ~name `Service multi_hash
   | `OCamlorg_v3c name -> pull_and_serve (module V3c_docker) ~name `Service multi_hash
+  | `Dune_distribution_docker name -> pull_and_serve (module Dune_distribution_docker) ~name `Service multi_hash
```

Comment:
```suggestion
  | `Dune_binary_distribution -> pull_and_serve (module Dune_binary_distribution) ~name `Service multi_hash
```

Reply:
We should stick with the _docker naming convention here.